src/nft/components/details/detailsV2/DataPageTraits.tsx

```diff
@@ -0,0 +1,19 @@
+import { Trans } from '@lingui/macro'
+
+import { Tab, TabbedComponent } from './TabbedComponent'
+
+const TraitsContent = () => {
+  return <div>Traits</div>
+}
+
+const DescriptionTabs: Array<Tab> = [
```

Comment:
Traits component doesn't have tabs it just has the one main page. We wouldn't want the Traits title to appear to the user as clickable